src/components/PhonicIcon/PhonicIcon.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../pages/Index/index.scss";
+.PhonicIcon {
```

Comment:
pls put like 1 line of whitespace between the import and between each of the classes